### ISSUE TITLE
Full-Text Search support for SQLite

### ISSUE THREAD

**Issue by matthewmueller** [prisma/prisma#9414]
_(follow-up from #1684)_

## Problem

In [2.30.0](https://github.com/prisma/prisma/releases/tag/2.30.0), we shipped Full-Text Search support for Postgres in Preview.

We have plans to also support [SQLite full-text search](https://www.sqlite.org/fts5.html). Some of the prerequisites for this feature is being able to manage indices in Migrate and Introspection and also compile the extension into SQLite.

## Suggested solution

It should have the [same Full-Text Search API](https://www.prisma.io/docs/concepts/components/prisma-client/full-text-search).

**Comment by Secretmapper** [comment]
Anyone found a way to do this manually through raw SQL while waiting for this to ship? How can we add migrations for this manually?

**Comment by jackbravo** [comment]
@Secretmapper, yes. You can do something like what is reported on #8106. Where you add custom SQL code to your database, adding full text indexes on sqlite. The problem, as reported on #8106 is that the shadow tables created by SQLite are not ignored when runngin `prisma migrate dev`. What I've done in a personal project is to run this full text index creation code outside of the migration steps. When I want to create a new migration I do:

1. update the prisma schema code with the changes I want
2. in my dev environment I run `prisma migrate reset`, to reset the database to not use my custom full text index.
3. then run `prisma migrate dev` to generate the new migration
4. run again my custom SQL to generate the full text index
5. if all test go well, deploy code to prod, but on prod I only run `prisma migrate deploy`

I hope this makes sense

**Comment by benjaminknauer** [comment]
Any timeline for this?

**Comment by janpio** [comment]
No. We will put this on our [roadmap](https://pris.ly/roadmap) when we are ready to look into this. Right now other things are higher priority for us.

**Comment by colinricardo** [comment]
👀

**Comment by dubem-design** [comment]
👀

**Comment by shmuli9** [comment]
👀

**Comment by SiavoshZarrasvand** [comment]
> No. We will put this on our [roadmap](https://pris.ly/roadmap) when we are ready to look into this. Right now other things are higher priority for us.

Any update on this? Been over two years. 
For me personally, I've stopped using Prisma due to this.

**Comment by colinricardo** [comment]
👀

**Comment by bramses** [comment]
Would love an update or even a workaround for this, if anyone has done anything

**Comment by SiavoshZarrasvand** [comment]
> Would love an update or even a workaround for this, if anyone has done anything

Workaround: Use another DB. DuckDB isn't too bad, and is in-memory as well. https://duckdb.org/docs/extensions/full_text_search.html

**Comment by kr05** [comment]
Hello! Thanks for all the work done on Prisma. Are there still plans on implementing FTS for sqlite?

**Comment by SiavoshZarrasvand** [comment]
> Hello! Thanks for all the work done on Prisma. Are there still plans on implementing FTS for sqlite?

Highly doubt it mate. Seems to be 0 progression for years and not even replying here.
